### DISCUSSION TITLE
Is it 64 bit?

### DISCUSSION THREAD

**Discussion by leontievle** [PaulWoitaschek/Voice#1597] [Q&A] [answered]
The new Google phones Pixel 7 and Pixel 7 Pro are 64-bit only. Will the Voice application work there?

**Comment by xcy7e** [comment, marked as answer]
Of course it will.
Afaik it's built for arm64 architecture.

I love to hear that people choose their devices based on if Voice runs on it.
I share your love !

**Reply by PaulWoitaschek** [reply]
Yep.

Written from my Pixel 7 😁
